docs/source/azure/advanced/bloboutput.rst

```diff
@@ -9,9 +9,18 @@ Writing Custodian Output to Azure Blob Storage
 You may pass the URL to a blob storage container as the output path to Custodian.
 You must change the URL prefix from https to azure.
 
+By default, Custodian will add the policy name and date as the prefix to the blob.
+
     .. code-block:: sh
 
         custodian run -s azure://mystorage.blob.core.windows.net/logs mypolicy.yml
 
+In addition, you can use `pyformat` syntax to format the output prefix.
+This is example is the as same structure as the default one.
```

Comment:
nit: should be 'This example is the same structure as the default one' (remove extra is and as)